dockerfiles/startall.sh

```diff
@@ -87,7 +88,7 @@ while [ "$#" -gt 0 ]; do
          echo "    -c                 Update chart dependencies"
          echo "    -p <location>      Use local persistent data"
          echo "    -pv <yamlfile>     Override dataplane definitions for local persistent data"
-         echo "    -e                 Deploy light-weight Elastic Stack for component log processing"
+         echo "    -e                 Ignored - light-weight Elastic Stack deployed by default"
```

Comment:
Would it be sensible to repurpose -e to suppress the deployment of elk?

Reply:
Same question on the uninstall?

Reply:
The stopall attempts to uninstall the elastic cluster regardless, no need to include the option.
That's based on precedent behavior encouraged by Jake/Gavin in other PRs related to stopall.